### DISCUSSION TITLE
Meter with battery (read only a few times)

### DISCUSSION THREAD

**Discussion by dpunzeng** [jomjol/AI-on-the-edge-device#1915] [Ideas] [answered]
Hi all, I have a few systems running very nicely. A friend has a meter box with no power available.
I have used low power sleep mode with esp8266 in other projects.

Is there anything, that hinders us to do this with this project? 
Having a reading e few times a day or even only once a day seems to be sufficient in a lot of cases.

Any input welcome.

Best Dieter

**Comment by caco3** [comment, marked as answer]
This was asked before.

The ESP32-CAM is not really made for low power, thus there is not much we could optimize by software.

See https://github.com/jomjol/AI-on-the-edge-device/discussions/1265

**Reply by 1337gut** [reply]
I can't agree to that. I've recently installed a [timelapse cam](https://github.com/raduprv/esp32-cam_ov2640-timelapse) based on the ESP32-CAM and it uses deep sleep. This way I can take pictures for several days.

**Reply by caco3** [reply]
well, if it works, it would be great.
@jomjol had a look on it once, see https://github.com/jomjol/AI-on-the-edge-device/issues/234

If you are willing to look into implementing it, we would be happy for support!
